cmd/kubelet/app/options/options.go

```diff
@@ -494,6 +494,7 @@ func AddKubeletConfigFlags(mainfs *pflag.FlagSet, c *kubeletconfig.KubeletConfig
 	fs.Int32Var(&c.EventBurst, "event-burst", c.EventBurst, "Maximum size of a bursty event records, temporarily allows event records to burst to this number, while still not exceeding event-qps. Only used if --event-qps > 0")
 
 	fs.BoolVar(&c.EnableDebuggingHandlers, "enable-debugging-handlers", c.EnableDebuggingHandlers, "Enables server endpoints for log collection and local running of containers and commands")
+	fs.BoolVar(&c.EnableProfiling, "profiling", c.EnableProfiling, "DEPRECATED: enable profiling via web interface host:port/debug/pprof/")
```

Comment:
Why create a Deprecated flag in the first place? Why not just use the config file?

Reply:
a careless copy-pasting，fixed it